specs/order_spec.rb

```diff
@@ -36,15 +45,32 @@
     end
   end
 
-  describe "#add_product" do
+  xdescribe "#add_product" do
     it "Increases the number of products" do
-      products = { "banana" => 1.99, "cracker" => 3.00 }
+      products = {"banana" => 1.99, "cracker" => 3.00 }
       before_count = products.count
       order = Grocery::Order.new(1337, products)
 
       order.add_product("salad", 4.25)
       expected_count = before_count + 1
-      order.products.count.must_equal expected_count
+      #returns true or false
+      order.products.count == expected_count
+    end
+
+     #tests remove product method
+    xdescribe "#remove_product" do
+      it "Decreases the number of products" do
+        #arrange
```

Comment:
Tests for `remove_product` should not be inside the `describe` block for `add_product` - they should probably have their own `describe`.

You're also missing an interesting test case for `remove_product`: what if the product isn't in the order?